manifests/init.pp

```diff
@@ -10,14 +10,16 @@
   Boolean $debug = false,
   Boolean $trace = false,
   Boolean $announce_cluster = false,
-  String $binpath = "/usr/sbin/gnatsd",
-  String $configdir = "/etc/gnatsd",
-  String $piddir = "/var/run",
+  Stdlib::Absolutepath $binpath = "/usr/sbin/gnatsd",
+  Stdlib::Absolutepath $configdir = "/etc/gnatsd",
+  Stdlib::Absolutepath $piddir = "/var/run",
   String $binary_source = "puppet:///modules/nats/gnatsd-0.9.4",
   String $service_name = "gnatsd",
-  String $cert_file = "/etc/puppetlabs/puppet/ssl/certs/${facts['networking']['fqdn']}.pem",
-  String $key_file = "/etc/puppetlabs/puppet/ssl/private_keys/${facts['networking']['fqdn']}.pem",
-  String $ca_file = "/etc/puppetlabs/puppet/ssl/certs/ca.pem",
+  Enum['running', 'stopped'] $service_ensure = 'running',
```

Comment:
Yes I don't care for puppet lint it's got some really old ideas. Many other languages have updated their style guides regarding quotes because it's easier to be consistent with doubles